docs/deployment/minikube.md

```diff
@@ -58,7 +58,9 @@ Kong is accesssible. This will be used to actually send reqeusts into the
 Kubernetes cluster.
 
 ```bash
-$ export PROXY_IP=$(minikube service -n kong kong-proxy --url | head -1)
+$ export PROXY_IP=$(minikube service -n kong kong-kong-proxy --url | head -1)
+# If installed by helm without specifying namespace, use the line below instead.
+# $ export PROXY_IP=$(minikube service kong-kong-proxy --url | head -1)
```

Comment:
I'd advise to educate the user to figure out the service and command rather than providing this command.   This command assumes that you do `helm install kong/kong --name kong`. If a user doesn't provide the name `kong`, then the service name will be different.

Reply:
I see. How about letting users know this?
As you mentioned, yes, people can figure out the service name by themselves. 
But if the docs note the relationship between helm release name and service name, it'd be more explicit.

```bash
# If installed by helm without specifying namespace, use the line below instead.
# $ export PROXY_IP=$(minikube service <release-name>-kong-proxy --url | head -1)
```

Reply:
The docs are meant for YAML based installation. We can possibly add a note that if you are using Helm, you need to figure the service name by yourself.

Reply:
@hbagdi 

> The docs are meant for YAML based installation. We can possibly add a note that if you are using Helm, you need to figure the service name by yourself.

Of course saying "You can find your service name by yourself" might be good, I thought mentioning an explicit name resolution like "Service name would be `<release-name>`-kong-proxy" would be more convenient.

I added a new commit. I would be thankful if you can review it.